src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java

```diff
@@ -617,6 +616,12 @@ private void updateLanguage(CompletionManager completionManager)
       if (fileType_ == null)
          return;
 
+      if (completionManager_ != null)
```

Comment:
Do we need to do any of this attach/detach when the editor itself is attached/detached? (cf. AceEditor.java#410)